lisa/sut_orchestrator/azure/common.py

```diff
@@ -1444,13 +1447,19 @@ def load_environment(
     for node in environment.nodes.list():
         assert isinstance(node, RemoteNode)
 
+        vm = vms_map.get(node.name)
+        assert_that(vm).is_not_none()
```

Comment:
Add error message, so it's easy to know what's wrong with the assertion failure.